pkg/manifest/service.go

```diff
@@ -54,6 +55,13 @@ type Service struct {
 	AccessControl      AccessControlOptions  `yaml:"accessControl,omitempty"`
 }
 
+type Annotations []Annotation
+
+type Annotation struct {
+	Key   string      `yaml:"key"`
+	Value interface{} `yaml:"value"`
```

Comment:
why this is interface instead of string?

Reply:
The value could be of a string as well as map, so needed to keep this as interface to handle nested json structure.